spark/processing/3.3/py3/docker/py39/Dockerfile.cpu

```diff
@@ -0,0 +1,132 @@
+FROM 137112412989.dkr.ecr.us-west-2.amazonaws.com/amazonlinux:2
+ARG REGION
+ENV AWS_REGION ${REGION}
+
+RUN yum clean all \
+    && yum update -y \
+    && yum install -y awscli bigtop-utils curl gcc gzip unzip zip gunzip tar wget liblapack* libblas* libopencv* libopenblas*
+
+# Install python 3.9
+ARG PYTHON_BASE_VERSION=3.9
```

Comment:
Is this the latest version supported by emr release?

Reply:
As discussed offline, retaining the current python version (python 3.9) as per the plan.